### ISSUE TITLE
Lists itself as a dependency for all plugins

### ISSUE THREAD

**Issue by ParticularlyPythonicBS** [demystifying-javascript/python-extension-pack#1]
This pack is dangerous and can ruin setups.
If you need to uninstall this pack, you will need to uninstall all the packages that come along with it whether it was installed through the package or not.
This is because the package lists itself as a dependency for all the plugins it installs.

**Comment by ayush-sarda-rpf** [comment]
exactly. I accidentally installed it and to uninstall, I have to uninstall all the other dependencies.

**Comment by hunterMG** [comment]
Some extensions in this pack keep creating `.zcompdump-{hostname}-{zsh-version}.{hostname}.local.{PID}` files in the home folder every time VSCode starts.
It creates a zsh shell when VSCode starts, which seems suspicious.
